graphql-cli/src/main/java/io/ballerina/graphql/cmd/Utils.java

```diff
@@ -109,28 +113,97 @@ public static GraphQLSchema getGraphQLSchemaDocument(String schema, Extension ex
             String sdlInput = extractSchemaContent(schema);
             typeRegistry = schemaParser.parse(sdlInput);
         }
-        GraphQLSchema graphQLSchema = schemaGenerator.makeExecutableSchema(typeRegistry,
-                RuntimeWiring.MOCKED_WIRING);
+        GraphQLSchema graphQLSchema = schemaGenerator.makeExecutableSchema(typeRegistry, RuntimeWiring.MOCKED_WIRING);
+        return graphQLSchema;
+    }
+
+
+    /**
+     * Returns the `GraphQLSchema` instance for a given Federated GraphQL schema file or schema URL.
+     *
+     * @param schema     the schema value of the Graphql config file
+     * @param extensions the extensions value of the Graphql config file
+     * @return the `GraphQLSchema` instance
+     * @throws IntospectionException If an error occurs during introspection of the GraphQL API
+     * @throws SchemaProblem         If a GraphQL schema related error occurs
+     * @throws IOException           If an I/O error occurs
+     */
+    public static GraphQLSchema getGraphQLFederatedSchemaDocument(String schema, Extension extensions)
```

Comment:
```suggestion
    public static GraphQLSchema getGraphqlFederatedSchemaDocument(String schema, Extension extensions)
```

Reply:
Yes, if you have an abbreviated part in the name, use simple letters, unless it is the first letter of the word. 

E.g.: `httpVersion`, `getHttpVersion`, etc.